src/osvr/Util/LogRegistry.cpp

```diff
@@ -187,7 +205,7 @@ namespace util {
             // File sink - rotates daily
             std::string logDir;
             try {
-                size_t q_size = 1048576; // queue size must be power of 2
+                size_t q_size = 65536; // queue size must be power of 2
```

Comment:
It is critically important that the system not block rendering when flushing non-error messages to the screen -- it is okay to have a separate logging-consumer thread that blocks on the disk write after gobbling up all queued client-thread messages, but having the display burp every time we (for example) report update rate is not acceptable.

Almost as important is making the error messages appear when the program crashes.  I'd be willing to have error messages cause a rendering glitch in order to have this happen.  That is, asynchronous on info/warning and sync on error.

If things really flush correctly at crash, then all async should be okay.

One thing to consider is whether the OS keeps up with the background saving and never causes the buffer to fill.  If it fills and blocks the client thread to dump all of that to disk, we're going to get a large render blip.  Again, okay if that is a different thread but not okay if the render hangs due to logging.